### ISSUE TITLE
Cutting windows 7 support

### ISSUE THREAD

**Issue by igor-hnizdo** [trezor/trezord-go#273]
I will make a new issue for this.

Should we cut Windows 7?

It will greatly simplify the windows code (much less libusb custom patches, much less complex installation). But, windows 7 has still about 7% marketshare (compare to MacOS that has about 4-5 %).

**Comment by prusnak** [comment]
Will ask our support.

**Comment by igor-hnizdo** [comment]
oh nice!

...but I just realized the custom libusb code will need to still be there even on Windows 10, because it will show the device and try to connect to it even _while the device is still installing_ . Now I remember it was that issue even on Windows 10 when Trezor is first connected.

So I will unfortunately need to keep the code there. But I can remove the installation of libwdi driver, as it is disabled on Windows 10.

**Comment by igor-hnizdo** [comment]
I was able to find the commits where we added the winusb custom code originally.

https://github.com/trezor/trezord-go/commit/f141e57ea219664b04812aff559e84621a288970
https://github.com/trezor/trezord-go/commit/dad2e755c976f9ed8926aacd8b683fe310a23419

Seems it's just for Windows 7. BUT, I am a bit too afraid to touch what works without really extensive testing on Windows 10, which I cannot do right now. So let's just drop the libwdi stuff. And maybe the driver deletion with devcon, that's also a bit stupid and just for Windows 7.
